jazz_apigee-proxy-aws/components/validation.js

```diff
@@ -0,0 +1,39 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================const errorHandlerModule = require("./components/error-handler.js");
+
+const errorHandlerModule = require("./error-handler.js");
+
+module.exports.validateEvent = (event, callback) => {
+    const errorHandler = errorHandlerModule();
+    if (!event.region) {
+        callback(JSON.stringify(errorHandler.throwMissingParamsError(`Required parameter 'region' is missing`)));
+    }
+    if (!event.body) {
+        callback(JSON.stringify(errorHandler.throwMissingParamsError(`Required parameter 'body' is missing`)));
+    }
+    if (!event.queryStringParameters) {
```

Comment:
Same as above, I don't think that we need a required check on event.queryStringParameters (ex: POST /api?)